### PR TITLE
Step 2 – reading web3 near certified developer level 1

### PR DESCRIPTION
**Pull request by shepf**

### CODE REVIEW

File: cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+## 步骤2 – 阅读web3（NEAR开发者L1认证）
+
+学习阅读控制资金，身份和所有权的合约。adfa
```
Comment (tigerjibo) [inline cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md:3]:
```suggestion
学习阅读关于控制资金，身份和所有权的合约
```

**Review by tigerjibo — COMMENTED** [review]
review

File: cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
+## 步骤2 – 阅读web3（NEAR开发者L1认证）
+
+学习阅读关于控制资金，身份和所有权的合约
+
+> 今天的目标是阅读尽可能多的合约。
+
+有时人们不确定如何完成此活动。 如下技巧可能会有所帮助。
+
```
Comment (MartinYeung5) [inline cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md:8]: 可以考慮多次快速閱讀而不是一次性的慢慢閱讀。閱讀[How to survey?](https://hackmd.io/@nearly-learning/how-to-survey) 可以讓你明白為什麼和怎樣去做。

File: cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+尝试弄清楚如何获得合约代码。 然后快速阅读代码。 希望您尽可能多的阅读合约代码。 探索这些合约的“目的”（why is it?）和“编码”（what is it?）
+
+* AssemblyScript合约 是以.ts结尾的文件，通常是main.ts或index.ts
+* Rust合约 始终位于名为lib.rs的文件中。
```
Comment (MartinYeung5) [inline cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md:12]:
```suggestion
* Rust合约 位于名为lib.rs的文件中。
```

File: cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md (hunk 2 above)
Comment (MartinYeung5) [reply cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md:8]: 補充一下，
原文有一段
"Consider that it is better to read many times quickly and NOT just one time slowly. Read How to survey? to better understand why and how you can do this"

File: cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+有时人们不确定如何完成此活动。 如下技巧可能会有所帮助。
+
+尝试弄清楚如何获得合约代码。 然后快速阅读代码。 希望您尽可能多的阅读合约代码。 探索这些合约的“目的”（why is it?）和“编码”（what is it?）
+
+* AssemblyScript合约 是以.ts结尾的文件，通常是main.ts或index.ts
+* Rust合约 位于名为lib.rs的文件中。
+
+核心活动：
+1. 在下面的资源部分中，选择AssemblyScript或Rust
+* 查看您的语言标记为“CORE Activity”的3份合约
```
Comment (MartinYeung5) [inline cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md:16]:
```suggestion
* 查看您所选的语言中标记为“CORE Activity”的3份合约
```

File: cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+奖励活动：
+如果您还有时间去探索，那么您可以进行以下活动。
+clone所选语言的 starter 库并按照说明进行操作
```
Comment (MartinYeung5) [inline cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md:20]:
```suggestion
複製(clone)你所选语言的 starter 库并按照说明进行操作
```

File: cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+* [AssemblyScript](https://github.com/Learn-NEAR/starter--AssemblyScript)
+* Rust (仍在开发中)
+
+您阅读得越多越好，如果可以的话，继续阅读下去。记住要把注意力集中在阅读上，避免一些有关构建和运行上的问题的诱惑，因为这会分散您的注意力。明天您好就可以构建和运行。
```
Comment (MartinYeung5) [inline cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md:24]:
```suggestion
您阅读得越多越好，如果可以的话，继续阅读下去。记住要把注意力集中在阅读上，避免一些有关构建和运行上的问题的诱惑，因为这会分散您的注意力。明天，您就可以构建和运行所有东西。
```

File: cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+
+1. 阅读您所选择的语言的所有合约，而不仅仅是3种
+2. 还要阅读其他语言的所有合约。
+3. 查看这些小难题，它们可能对您很有趣
```
Comment (MartinYeung5) [inline cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md:28]:
```suggestion
3. 查看这些小难题，您可能会在它们中获得乐趣
```

File: cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+
+* [Scavenger Hunt Challenge #1](https://hackmd.io/@nearly-learning/hunt-01)
+* [Scavenger Hunt Challenge #2](https://hackmd.io/@nearly-learning/hunt-02)
+
+
+更加深入的探索
+如果您感到无所畏惧，那么您可能会在一天中做完这些
+
+* 查看所有[核心合约](https://github.com/near/core-contracts) 和观看在线代码审计[视频](https://www.youtube.com/watch?v=BA7VeUS_RAA&list=PL9tzQn_TEuFXnYksuNJwrl1l_AuWzn6eF)
+* 审计[Illia的合约仓库](https://github.com/ilblackdragon/contracts)
```
Comment (MartinYeung5) [inline cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md:38]:
```suggestion
*  查看[Illia的合约仓库](https://github.com/ilblackdragon/contracts)
```

File: cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+* 审计所有的彩虹桥黑客马拉松合约
```
Comment (MartinYeung5) [inline cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md:39]:
```suggestion
* 查看所有的彩虹桥黑客马拉松合约
```

File: cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md [hunk 10]
```diff
@@ -0,0 +1,92 @@
...
+3. 查看这些小难题，它们可能对您很有趣
+
+* [Scavenger Hunt Challenge #1](https://hackmd.io/@nearly-learning/hunt-01)
+* [Scavenger Hunt Challenge #2](https://hackmd.io/@nearly-learning/hunt-02)
+
+
+更加深入的探索
+如果您感到无所畏惧，那么您可能会在一天中做完这些
+
+* 查看所有[核心合约](https://github.com/near/core-contracts) 和观看在线代码审计[视频](https://www.youtube.com/watch?v=BA7VeUS_RAA&list=PL9tzQn_TEuFXnYksuNJwrl1l_AuWzn6eF)
```
Comment (MartinYeung5) [inline cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md:37]:
```suggestion
* 查看所有[核心合约](https://github.com/near/core-contracts) 和观看相关的在线代码评论[视频](https://www.youtube.com/watch?v=BA7VeUS_RAA&list=PL9tzQn_TEuFXnYksuNJwrl1l_AuWzn6eF)
```

File: cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md (hunk 5 above)
Comment (shepf) [reply cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md:20]: 专业术语，这个不用改了吧

File: cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md [hunk 11]
```diff
@@ -0,0 +1,92 @@
...
+
+奖励活动：
+如果您还有时间去探索，那么您可以进行以下活动。
+clone所选语言的 starter 库并按照说明进行操作
+* [AssemblyScript](https://github.com/Learn-NEAR/starter--AssemblyScript)
+* Rust (仍在开发中)
+
+您阅读得越多越好，如果可以的话，继续阅读下去。记住要把注意力集中在阅读上，避免一些有关构建和运行上的问题的诱惑，因为这会分散您的注意力。明天您好就可以构建和运行。
+
+1. 阅读您所选择的语言的所有合约，而不仅仅是3种
```
Comment (MartinYeung5) [inline cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md:26]:
```suggestion
1. 阅读您所选择的语言的所有合约，而不仅仅是3份
```

File: cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md [hunk 12]
```diff
@@ -0,0 +1,92 @@
...
+记住：今天的活动
+
+* 您无需构建和测试每个合约
+* 您无需了解每一行代码
+* 您应该像看艺术或听音乐一样简单地阅读合约，而不必担心某个点或内容
+
+### AssemblyScript
+如果您打算专注于AssemblyScript，请[打开AssemblyScript合约列表](https://airtable.com/shrG4kGx80F55usI4/tblwK6r2kVUfS7gDn)
+
+您应该至少阅读**3个**标记为**CORE Activity**的合约。
```
Comment (MartinYeung5) [inline cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md:60]:
```suggestion
您应该至少阅读**3份**标记为**CORE Activity**的合约。
```

File: cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md [hunk 13]
```diff
@@ -0,0 +1,92 @@
...
+
+### Rust
+如果您打算专注于Rust，请[打开Rust合约列表](https://airtable.com/shrckdZAMgjbP3uBC/tblwK6r2kVUfS7gDn)
+
+您应该至少阅读**3个**标记为**CORE Activity**的合约。
```
Comment (MartinYeung5) [inline cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md:67]:
```suggestion
您应该至少阅读**3份**标记为**CORE Activity**的合约。
```

File: cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md (hunk 5 above)
Comment (shepf) [reply cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md:22]:
```suggestion
复制（clone）所选语言的 starter 库并按照说明进行操作
```

File: cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md [hunk 14]
```diff
@@ -0,0 +1,92 @@
...
+
+您可以阅读有关[Rust合约](https://hackmd.io/@nearly-learning/contract-basics-rust)基础的更多信息，也可以观看[视频](https://www.youtube.com/watch?v=AOvqsD-rBUw)。
```
Comment (MartinYeung5) [inline cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md:69]:
```suggestion
您可以阅读更多有关[Rust合约的基础知识](https://hackmd.io/@nearly-learning/contract-basics-rust)，也可以选择观看[视频](https://www.youtube.com/watch?v=AOvqsD-rBUw)以了解Rust合约。
```

File: cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md [hunk 15]
```diff
@@ -0,0 +1,92 @@
...
+* 0
+* >1
+
+现在，是时候提交任务了！  
+问题2:  请提交评论：
+
+合约1：
+* 合约的名字
+* 你对它感到兴趣的地方是什么
+* 理解这个合约哪一部分对您来说是挑战
```
Comment (MartinYeung5) [inline cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md:84]:
```suggestion
* 在阅读这个合约过程中，哪一部分对您来说是最具挑战
```

File: cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md [hunk 16]
```diff
@@ -0,0 +1,92 @@
...
+
+...
+
+合约N:
+* 合约的名字
+* 你对它感到兴趣的地方是什么
+* 理解这个合约哪一部分对您来说是挑战
```
Comment (MartinYeung5) [inline cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md:91]:
```suggestion
* 在阅读这个合约过程中，哪一部分对您来说是最具挑战
```

File: cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md [hunk 17]
```diff
@@ -0,0 +1,94 @@
...
+> 今天的目标是阅读尽可能多的合约。
+
+有时人们不确定如何完成此活动。 如下技巧可能会有所帮助。
+
+可以考虑多次快速阅读而不是一次性的慢慢阅读。阅读[How to survey](https://hackmd.io/@nearly-learning/how-to-survey) ?可以让您明白为什么和怎样去做。
+
+尝试弄清楚如何获得合约代码。 然后快速阅读代码。 希望您尽可能多的阅读合约代码。 探索这些合约的“目的”（why is it?）和“编码”（what is it?）
+
+* AssemblyScript合约 是以.ts结尾的文件，通常是main.ts或index.ts
+* Rust合约 位于名为lib.rs的文件中。
```
Comment (think-in-universe) [inline cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md:14]:
```suggestion
* Rust合约 位于名为`lib.rs`的文件中。
```

Comment (think-in-universe) [reply cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md:14]: 文件名可以采用 \`filename\`  语法标记，下同

File: cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md [hunk 18]
```diff
@@ -0,0 +1,94 @@
...
+
+核心活动：
```
Comment (think-in-universe) [inline cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md:16]: 1. heading 应添加格式
2. 丢失了 emoji

Comment (think-in-universe) [reply cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md:16]: 下同

**Review by think-in-universe — CHANGES_REQUESTED** [review]
翻译时，注意格式与原文匹配

**Comment by shepf** [comment]
fixed

File: cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md [hunk 19]
```diff
@@ -0,0 +1,94 @@
+## 步骤2 – 阅读web3（NEAR开发者L1认证）
+
+学习阅读关于控制资金，身份和所有权的合约
+
+> :sparkles: 今天的目标是阅读尽可能多的合约。:sparkles: 
+
+有时人们不确定如何完成此活动。 如下技巧可能会有所帮助。
```
Comment (danielwpz) [inline cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md:7]:
```suggestion
一开始可能会不知所措，如下技巧可能会帮助到你：
```

File: cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md [hunk 20]
```diff
@@ -0,0 +1,94 @@
...
+* AssemblyScript合约 是以.ts结尾的文件，通常是main.ts或index.ts
+* Rust合约 位于名为`lib.rs`的文件中。
+
+### :green_book: 核心活动：
+1. 在下面的资源部分中，选择AssemblyScript或Rust
+* 查看您所选的语言中标记为“CORE Activity”的3份合约
+
+### :blue_book: 奖励活动：
+如果您还有时间去探索，那么您可以进行以下活动。
+复制（clone）所选语言的 starter 库并按照说明进行操作
```
Comment (danielwpz) [inline cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md:22]:
```suggestion
克隆（clone）所选语言的 starter 库并按照说明进行操作
```

File: cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md [hunk 21]
```diff
@@ -0,0 +1,94 @@
...
+* [AssemblyScript](https://github.com/Learn-NEAR/starter--AssemblyScript)
+* Rust (仍在开发中)
+
+您阅读得越多越好，如果可以的话，继续阅读下去。记住要把注意力集中在阅读上，避免一些有关构建和运行上的问题的诱惑，因为这会分散您的注意力。明天，您就可以构建和运行所有东西。
```
Comment (danielwpz) [inline cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md:26]:
```suggestion
您阅读得越多越好，如果可以的话，继续阅读下去。记住要把注意力集中在阅读上，先不去考虑有关构建和运行上的问题，因为这会分散您的注意力。明天，您就可以构建和运行所有东西。
```

File: cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md [hunk 22]
```diff
@@ -0,0 +1,94 @@
...
+
+1. 阅读您所选择的语言的所有合约，而不仅仅是3份
+2. 还要阅读其他语言的所有合约。
+3. 查看这些小难题，您可能会在它们中获得乐趣
```
Comment (danielwpz) [inline cn/resources/ncd/level1/step-2-reading-web3-near-certified-developer-level-1.md:30]:
```suggestion
3. 看看下面这些小的挑战，您可能会在它们中获得乐趣
```

**Review by danielwpz — CHANGES_REQUESTED** [review]

**Comment by think-in-universe** [comment]
The PR is not ready actually, but I have to merge it first, and then fix the remaining issues.

**Comment by MartinYeung5** [comment]
翻译质量评价: 一般

**Comment by think-in-universe** [comment]
翻译质量评价：一般

**Pull request merged**